packages/@n8n/nodes-langchain/package.json

```diff
@@ -133,13 +133,13 @@
     "@aws-sdk/credential-provider-node": "3.549.0",
     "@getzep/zep-js": "0.9.0",
     "@google-ai/generativelanguage": "0.2.1",
-    "@google/generative-ai": "^0.5.0",
+    "@google/generative-ai": "0.7.1",
```

Comment:
looks like `0.8.0` was released yesterday. can we upgrade to that ?

Reply:
Makes sense. Updated & confirmed the node is still working as expected.